tests/mpi/renumber_cuthill_mckee.cc

```diff
@@ -22,6 +22,7 @@
 #include "../tests.h"
 #include <deal.II/base/logstream.h>
 #include <deal.II/base/mpi.h>
+#include <deal.II/base/mpi.templates.h>
```

Comment:
I do not really feel strongly about where these functions should live, but since they are internal the logical place seems to be the template file.